tests/ui_tests/test_jobs_page.py

```diff
@@ -81,57 +92,127 @@ def provider(provider_bundle: Bundle) -> Provider:
     return provider_bundle.provider_create('Awesome Provider')
 
 
-def _test_run_action(page: JobListPage, action_owner: Union[Cluster, Service, Provider, Host]):
-    """
-    Run the "Long" action
-    Check popup info
-    Check table info without filter (All)
-    Activate filter "Running"
-    Check table info
-    """
-    owner_name = action_owner.name if action_owner.__class__ != Host else action_owner.fqdn
-    expected_info = {
-        'status': JobStatus.RUNNING,
-        'action_name': LONG_ACTION_DISPLAY_NAME,
-        'invoker_objects': owner_name,
-    }
-    with allure.step(
-        f'Run action "{LONG_ACTION_DISPLAY_NAME}" on {action_owner.__class__}'
-    ), page.table.wait_rows_change():
-        long_action = action_owner.action(display_name=LONG_ACTION_DISPLAY_NAME)
-        long_action.run()
-    _check_job_info_in_popup(page, {'status': expected_info['status'], 'action_name': expected_info['action_name']})
-    _check_running_job_info_in_table(page, expected_info)
-    page.select_filter_running_tab()
-    _check_running_job_info_in_table(page, expected_info)
-
-
-@allure.step('Check running job information in table')
-def _check_running_job_info_in_table(page: JobListPage, expected_info: dict):
-    """Get info about job from table and check it"""
-    wait_and_assert_ui_info(
-        {**expected_info, 'start_date': is_not_empty, 'finish_date': is_empty},
-        page.get_task_info_from_table,
-    )
-
-
-@allure.step('Check finished job information in table')
-def _check_finished_job_info_in_table(page: JobListPage, expected_info: dict):
-    """Get and check info about successfully finished job from table"""
-    wait_and_assert_ui_info(
-        {**expected_info, 'start_date': is_not_empty, 'finish_date': is_not_empty},
-        page.get_task_info_from_table,
-    )
-
-
-@allure.step('Check job information in popup')
-def _check_job_info_in_popup(page: JobListPage, expected_info: dict):
-    """Get job info from popup and check it"""
-    with page.header.open_jobs_popup():
-        wait_and_assert_ui_info({**expected_info}, page.get_task_info_from_popup)
+@pytest.fixture()
+def created_hosts(provider: Provider) -> List[Host]:
+    """Create 11 hosts for "parallel" actions execution"""
+    return [provider.host_create(f'host-{i}') for i in range(11)]
+
+
+def test_run_multijob(cluster: Cluster, page: JobListPage):
```

Comment:
mb use Test class for this test funcs or specify name? Won't we mix up this func names in other files occasionally?

Reply:
I don't think any kind of problem will appear, but I moved test methods in class

Reply:
I don't think any kind of problem will appear, but I moved test methods in class